### ISSUE TITLE
[travis] Put all the language and compiler dependencies on a docker image

### ISSUE THREAD

**Issue by ryukinix** [DestructHub/ProjectEuler#55]
With this we reduce the time on build tests and we can use packages more updated. Remember that: travis use ubuntu 12.04 LTS. Very very legacy.

Maybe after that we can fix #13 

```
  - sudo apt-get update
  - sudo apt-get install elixir -y
  - sudo apt-get install php5 -y
  - sudo apt-get install golang -y
  - sudo apt-get install clojure -y
  - sudo apt-get install ghc -y
  - sudo apt-get install g++ gcc -y
  - sudo apt-get install lua5.2 -y
  - sudo apt-get install ruby-full -y
  - sudo apt-get install clisp -y
```

**Comment by lubien** [comment]
For better performance shouldn't it be best to push DestructHub's built image to Docker Hub?

Maybe you should create an account for us there.

**Comment by ryukinix** [comment]
Would be nice.

**Comment by ryukinix** [comment]
Implemented on #115 

We need just create an account for destructhub to close this issue.
